include/flatbuffers/idl.h

```diff
@@ -536,6 +536,9 @@ struct ServiceDef : public Definition {
 
 // Container of options that may apply to any of the source/text generators.
 struct IDLOptions {
+  // field case style options for C++
+  enum Case { Case_Unchanged = 0, Case_Upper, Case_Lower };
```

Comment:
The `Case` name doesn't say anything about this enum. It would be better to choose a more informative name.

Reply:
will rename to CaseStyle